docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
+data to keep an account's encrypted content secure. In other words, the special status of providing
+a network sync for client data does not enable a server, or any entity between the server and
```

Comment:
I don't think this goes far enough. It should also include that the server cannot inject new data that the application presents as being the user's, decrypting successfully.

Reply:
I'm not sure I agree here, yet. It's less that ideal, but not the same as weakening the security of the content.

For example, would duplication of a single vault item be injecting new data? that feels more like a corruption event than a weakening of security. Of course, there might be scarier attacks like duplicating URIs between items to try and take advantage of autofill behaviors.

Then there's just totally new data that was never in the vault. that seems clearly in scope, but I don't know how to differentiate from the above.

I'd like to add something here, but I don't have the phrasing for it, yet.

Reply:
I agree that: injecting *net new, chosen plaintext* data (through e.g. fake orgs) should be covered here. That should never be possible to happen. Duplicating existing data is not an issue.

I don't know what you mean by:

> Then there's just totally new data that was never in the vault. that seems clearly in scope, but I don't know how to differentiate from the above.

"It should not be possible to inject chosen data into the user's decrypted vault.".

Reply:
that quote means 

> injecting net new, chosen plaintext data (through e.g. fake orgs) should be covered here

😅 

I'll update to include chosen cleartext data protection